src/GeohashHilbert.jl

```diff
@@ -108,15 +108,30 @@ string.
 - `lat::Real`: latitude of point to encode. Must be in [-90,90].
 - `precision::Integer`: precision of returned geohash. Must be positive.
 - `bits_per_char ∈ [2,4,6]`: how many bits of information each character encodes.
+
+!!! note
+    To avoid overflow in machine arithmetic, `precision * bits_per_char` must be <= 62.    This still allows meter-level granularity. If your use case requires sub-meter
```

Comment:
Should this be split into two lines?

Reply:
yes whoops. Atom wraps lines so prettily I didn't notice.